sdk/core/Azure.Core.Experimental/api/Azure.Core.Experimental.net461.cs

```diff
@@ -191,17 +192,77 @@ public partial struct DynamicJsonOptions
         public string Name { get { throw null; } }
         public Azure.Core.Dynamic.DynamicJson Value { get { throw null; } }
     }
+    public abstract partial class ObjectDocument : System.IDisposable
+    {
+        protected ObjectDocument() { }
+        public abstract T As<T>(object element);
```

Comment:
do these method need to be public or they could be protected internal?

Reply:
I had not considered the latter, but yes, that should work.